README-DEVELOPER.md

```diff
@@ -67,7 +67,11 @@ One can check coding style using
 ``` sh
 npm run lint
 ```
-Install the recommended extensions to automatically fix linting errors when possible.
+Install the recommended extensions to automatically fix linting errors through VSCode.
+You will also need to install the `eslint` dependencies globally:
```

Comment:
Are you sure? On my side this is not installed globally.

Reply:
Is eslint working live when you edit the code through VSCode?
I did need to install all this globally, otherwise it didn't work. You should see something like:
![image](https://github.com/user-attachments/assets/10a77d3a-fe02-444a-8496-7fb7644b3198)

Reply:
Or maybe you have a vscode-eslint configuration that allows to use the workspace's npm environment?

Reply:
It uses the one in the workspace's node_modules. If I delete that folder, eslint won't work until I run `npm install`. Quite sure this is not something I configured myself.

It seems all my eslint settings are on default

Reply:
![image](https://github.com/user-attachments/assets/cc6fb873-3423-492b-adec-3036fea62772)
Live linting is supposed to be a feature from ESLint VSCode extension. The linting from the command line is working fine, but the extension is not linting live. Installing eslint globally solves the issue, so I suspect there is something wrong with the config or vscode settings.

Reply:
Added a fix to use the workspace's node env in our settings.json

Reply:
This works for me too. but I also want to point out that the ESlint extension can live lint another project I have been working on which doesn't have `nodePath` set up and neither is the ESlint installed globally.